drivers/wifi/nxp/Kconfig.nxp

```diff
@@ -314,43 +314,69 @@ config NXP_WIFI_SHELL
 
 menu "Wi-Fi driver Stack configurations"
 
-config NXP_WIFI_MON_TASK_STACK_SIZE
+config NXP_WIFI_MON_STACK_SIZE
```

Comment:
Renaming Kconfigs are not something that can be done as they are already established in the project. 

Please revert this aspect of your PR to the previous names.